### ISSUE TITLE
BCML broke BOTW

### ISSUE THREAD

**Issue by minayondaim3** [NiceneNerd/BCML#59]
[log.txt](https://github.com/NiceneNerd/BCML/files/4841297/log.txt)

I installed BCML 2.8.0 earlier today, after having already played the game for sometime earlier. I'd already installed the microsoft c++ packs. The mods I used, hyrule rebalance and improved savage lynel bow, seemed to merge/install correctly. When I try to start cemu, it crashes after loading shaders. I'm attaching my log.

I disabled any(and ALL) mods I thought would conflict, I even uninstalled BCML. Don't want it anymore if its gonna break the game. Unfortunately all I'm left with is a broken game, I can't even start BOTW. Crashes after loading shaders as if BCML is still installed. 

To be clear, BCML wouldn't work after install, after disabling what I thought were conflicting mods, and seems to have broken BOTW after uninstalling it.

**Comment by NiceneNerd** [comment]
@minayondaim3 Two questions:
1. Before you uninstalled BCML, did you try remerging, and did you make sure you had no graphic pack mods enabled except for community packs?
2. After uninstalling, have you removed/disabled the master BCML graphic pack?

**Comment by minayondaim3** [comment]
1. I tried remerging several times. Recap
After installing BCML, I installed improved savage lynel bow, then hyrule rebalance. At this point I didn't remerge anything, I just installed the mods. Cemu crashed after loading shaders. Thats where I'm at now. Remerging didn't help. Disabling other mods, community pack and too many options pack, didn't help. Here's a log with everything off except master bcml
[log.txt](https://github.com/NiceneNerd/BCML/files/4841371/log.txt)

2. No, I didn't delete any folders or files if thats what you mean. However deleting that folder after uninstalling doesn't help. BCML really broke the game. Whereas earlier it would at least give me the false hope of starting, by completing the shader load, as of this moment the game crashes before it even loads the shaders. Other games won't even load. This thing broke cemu, not just botw. 

This is extremely upsetting.

**Comment by NiceneNerd** [comment]
If deleting the BCML folder from your graphic packs didn't solve it, then there is a 100% chance that BCML is not the cause of the crashes you're having now. BCML doesn't have the ability to do anything at all to Cemu once the BCML graphic pack has been removed.

Moreover, the log shows that the crash happens before the game even starts, and thus before any of the BCML mod files actually take effect.

**Comment by minayondaim3** [comment]
No BCML is the issue. I got help in the cemu discord. The method I used to install bcml directly modifies game files. I probably should've used a new mod format (option 2) on this page https://gamebanana.com/tools/6624

OR used this page instead of that one, since I probably would've gotten better direction here. The issue was that first option directly modified the game apparently, and messed up the settings.xml 

Deleting the settings.xml helped with my other games, but BOTW is broken. I'm gonna need to dump it again.

**Comment by minayondaim3** [comment]
No BCML is the issue. I got help in the cemu discord. The method I used to install bcml directly modifies game files. I probably should've used a new mod format (option 2) on this page https://gamebanana.com/tools/6624

OR used this page instead of that one, since I probably would've gotten better direction here. The issue was that first option directly modified the game apparently, and messed up the settings.xml 

Deleting the settings.xml helped with my other games, but BOTW is broken. I'm gonna need to dump it again.
